code/modules/cargo/exports/organs.dm

```diff
@@ -0,0 +1,38 @@
+/datum/export/organ
+	include_subtypes = FALSE	//Centcom doesn't need organs from non-humans.
```

Comment:
But if this is black market that mean it isn't syndicate. because black market also forks to syndie gear etc etc